functional-test/src/test/groovy/org/rundeck/util/gui/pages/activity/ActivityPage.groovy

```diff
@@ -44,4 +45,10 @@ class ActivityPage extends BasePage implements ActivityListTrait{
         els By.xpath("//*[contains(text(),'${jobName}')]")
     }
 
+    static void validateFirstActivityRow(List<WebElement> activityList) {
```

Comment:
the term `validate` in the name of this function does not convey much meaning. What does it mean for the first activity raw to be valid or invalid? It depends on the context.

Also, `Exception`s (albeit overused in java) are reserved for something exceptional.
Having no activity is a perfectly valid state in our business logic. A boolean would make much more sense.

E.g. function along the lines of `boolean hasActivity(List<WebElement> activityList)`

Now, we can simplify this further employing groovy truthy logic
```
!!activityList
```

So, do we even need this function?